### ISSUE TITLE
Blender 2.80: Import GLB - KeyError: 'bpy_prop_collection[key]: key "" not found'

### ISSUE THREAD

**Issue by 0Curtain0** [KhronosGroup/glTF-Blender-IO#744]
**Describe the bug**
Blender 2.80 fails to import a **.GLB** file and produces the following error message:
```
Traceback (most recent call last):
  File "/home/user/Downloads/Blender/blender-2.80-linux-glibc217-x86_64/2.80/scripts/addons/io_scene_gltf2/__init__.py", line 528, in execute
    return self.import_gltf2(context)
  File "/home/user/Downloads/Blender/blender-2.80-linux-glibc217-x86_64/2.80/scripts/addons/io_scene_gltf2/__init__.py", line 549, in import_gltf2
    BlenderGlTF.create(self.gltf_importer)
  File "/home/user/Downloads/Blender/blender-2.80-linux-glibc217-x86_64/2.80/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_gltf.py", line 41, in create
    BlenderScene.create(gltf, scene_idx)
  File "/home/user/Downloads/Blender/blender-2.80-linux-glibc217-x86_64/2.80/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_scene.py", line 79, in create
    BlenderNode.create(gltf, node_idx, None)  # None => No parent
  File "/home/user/Downloads/Blender/blender-2.80-linux-glibc217-x86_64/2.80/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_node.py", line 169, in create
    BlenderNode.create(gltf, child_idx, node_idx)
  File "/home/user/Downloads/Blender/blender-2.80-linux-glibc217-x86_64/2.80/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_node.py", line 130, in create
    BlenderNode.create(gltf, child_idx, node_idx)
  File "/home/user/Downloads/Blender/blender-2.80-linux-glibc217-x86_64/2.80/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_node.py", line 130, in create
    BlenderNode.create(gltf, child_idx, node_idx)
  File "/home/user/Downloads/Blender/blender-2.80-linux-glibc217-x86_64/2.80/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_node.py", line 130, in create
    BlenderNode.create(gltf, child_idx, node_idx)
  [Previous line repeated 3 more times]
  File "/home/user/Downloads/Blender/blender-2.80-linux-glibc217-x86_64/2.80/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_node.py", line 124, in create
    BlenderSkin.create_armature(gltf, pynode.skin_id, parent)
  File "/home/user/Downloads/Blender/blender-2.80-linux-glibc217-x86_64/2.80/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_skin.py", line 46, in create_armature
    obj.parent = bpy.data.objects[gltf.data.nodes[parent].blender_object]
KeyError: 'bpy_prop_collection[key]: key "" not found'

location: <unknown location>:-1
```

**To Reproduce**
1. Download .GLB file from here: https://www.deviantart.com/ssingh511/art/Into-the-Spiderverse-OBJ-GLB-815436959
2. Extract the .zip file.
3. Open Blender 2.80
4. Click on **File** - **Import** - **glTF 2.0**
5. Import the **Gwen85Jump.glb** file from the **Spider Gwen ITSV** folder.
6. Error message should display.

**Expected behavior**
This is my first time using GLB in Blender, so I was expecting to see a rigged 3D character model imported into the scene along with playable animations.

**.blend file/ .gltf**
Here is the link to the GLB file: https://www.deviantart.com/ssingh511/art/Into-the-Spiderverse-OBJ-GLB-815436959

**Version**
 - OS: Pop!_OS 64bit (Ubuntu Linux based)
 - Blender Version: 2.80

**Comment by julienduroure** [comment]
Duplicate of #700 

File has multiple skin with exactly same joint list, and some other skin with joint list that includes some other joint list of other skin.

**Comment by julienduroure** [comment]
Closed by #857
